docs/7_OngoingWork.md

```diff
@@ -0,0 +1,328 @@
+# Transfer learning and fine-tuning
+
+## The objective 
+The objective of this part of the project, is to use a model tht already exist, and to use it's already intelligent structure to use it to do something else.
+
+After the benchmark, we decided that MobileNetv3Large was the most adapted model for our needs. MobileNet is a model that has 1000 classes, so it can recognize 1000 different objects. In our case, we want it to be able to detect people in distress/wounded and people that are safe. So only 2 classes. 
+
+We chose to use leafs to represent people, for there are few datasets available of wounded people.
+
+
+## Fine-tuning with Pytorch
+
+During the last days of the project, we manage to finetune MobileNetv3Large with pytorch. After getting the finetuned model, we managed to make it run on the Raspberry Pi. 
+
+The model had poor results, but the POC is here, and your mission, if you accept it, is to improve and adjust the training.
+
+### Training
+
+First, you will have to download the script files that are in `misc/finetuning/finetuning.py` on the repo on the machine you want to do the training. They might be :  [here](https://www.youtube.com/watch?v=dQw4w9WgXcQ) 
+
+
+Once you have enjoyed downloading them, you will have to create a virtual environnement :
+```
+python3 -m venv venv
+source venv/bin/activate
+```
+
+Then install the dependencies :
+```
+pip install torch torchvision tqdn
+```
+And finally start the training in the background :
+```
+nohup python3 finetuning.py > training.log &
+``` 
+
+*N.B. :
+If you are connected with ssh on the machine where the training is happening, you can disconnect yourself, and check the results in real time on the training.log file*
+
+At the end of the training, you will have a .pth file : this is your fine tuned model.
+
+
+### Running the model on Raspberry Pi
+
+First, transfer the model from the machine you made the training to the Raspberry Pi. If you are using a remote server, use rsync from the raspberry pi : 
+```
+rsync -Pru login@ip:/path/to/the/model .
+```
+
+Then once you have it, same as before, you create your virtual environnement :
+```
+python3 -m venv venv
+source venv/bin/activate
+```
+
+Then install the dependencies :
+```
+pip install torch torchvision tqdn
+```
+
+Once it's done, you can download the script files that are in `misc/finetuning/objectRecognition.py` on the repo, and start the script (assuming you configured correctly your Raspberry Pi).
+
+
+Okay, so now you have a script that give's you a prediction of what the camera see's with a certitude percentage.
+
+### What's left
+
+#### The training
+
+The model, as mentionned before, has poor results. We didn't spend much time on improving the training, but this is something you can do. 
+
+I would advise that you don't rely too much on the scripts that were made. There were just POCs, and can be largely improved.
```

Comment:
Here, link to the official doc here https://pytorch.org/tutorials/beginner/transfer_learning_tutorial.html